extensions.json

```
@@ -900,8 +900,8 @@
     {
       "id": "platformio.platformio-ide",
       "repository": "https://github.com/platformio/platformio-vscode-ide",
-      "version": "1.10.0",
-      "checkout": "v1.10.0"
+      "version": "2.1.0",
+      "checkout": "v2.1.0"
```

When using `"download"` instead of `"repository"` (see suggestion above), we no longer need to use `"checkout"`:

```suggestion
      "version": "2.1.0"
```